source/appModules/soffice.py

```diff
@@ -220,8 +220,13 @@ def _get_states(self):
 			# #8988: Cells in Libre Office do not have the selected state when a single cell is selected (i.e. has focus).
 			# Since #8898, the negative selected state is announced for table cells with the selectable state.
 			states.add(controlTypes.STATE_SELECTED)
-		if self.IA2Attributes.get('Formula'):
-			# #860: Recent versions of Calc expose has formula state via IAccessible 2.
+		# #860: Recent versions of Calc expose has formula state via IAccessible 2.
+		# #10759: However in Libre Office 6.4 the attribute contains "!!br0ken!!"
+		# when there is no formula in the cell.
+		if (
+			self.IA2Attributes.get('Formula')

```

Comment:
Could you fetch the value before the if check, storing it in a variable, and then use the variable in both checks please.

Reply:
@michaelDCurran Done